ui/helpers/taskcluster.js

```diff
@@ -46,4 +46,19 @@ const taskcluster = (() => {
   };
 })();
 
+export const getAction = (actionArray, actionName) => {
+  // const action = actionArray.find(result => result.name === actionName);
+  const action = null;
```

Comment:
I assume this was added for debugging? :)

Reply:
Gah...  Yes, exactly.  Fixed.  Thanks.  :)